resources/route53-resolver-endpoints.go

```diff
@@ -0,0 +1,90 @@
+package resources
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/route53resolver"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+// Route53ResolverEndpoint is the resource type for nuking
+type Route53ResolverEndpoint struct {
+	svc  *route53resolver.Route53Resolver
+	id   *string
+	name *string
+	ips  []*route53resolver.IpAddressUpdate
+}
+
+func init() {
+	register("Route53ResolverEndpoints", ListRoute53ResolverEndpoints)
+}
+
+// ListRoute53ResolverEndpoints produces the resources to be nuked
+func ListRoute53ResolverEndpoints(sess *session.Session) ([]Resource, error) {
+	svc := route53resolver.New(sess)
+
+	params := &route53resolver.ListResolverEndpointsInput{}
+
+	resources := make([]Resource, 0)
+	output, err := svc.ListResolverEndpoints(params)
+
+	if err != nil {
+		return resources, err
+	}
+
+	for _, endpoint := range output.ResolverEndpoints {
+		resolverEndpoint := &Route53ResolverEndpoint{
+			svc:  svc,
+			id:   endpoint.Id,
+			name: endpoint.Name,
+		}
+
+		ipsOutput, err := svc.ListResolverEndpointIpAddresses(
+			&route53resolver.ListResolverEndpointIpAddressesInput{
+				ResolverEndpointId: endpoint.Id,
+			})
+
+		if err != nil {
+			return resources, err
+		}
+
+		for _, ip := range ipsOutput.IpAddresses {
+			resolverEndpoint.ips = append(resolverEndpoint.ips, &route53resolver.IpAddressUpdate{
+				Ip:       ip.Ip,
+				IpId:     ip.IpId,
+				SubnetId: ip.SubnetId,
+			})
+		}
```

Comment:
What do you actually need these for? They are not used in the output or remove function.

Reply:
Ah, left in from initial dev when we did have them in the output for a time.  I'll take them out, thanks!